.npmignore

```diff
@@ -0,0 +1,7 @@
+node_modules/generic-pool/.eslintignore
```

Comment:
Aren't the paths here wrong, shouldn't these be relative to the project root?

Reply:
errm.., yes - those look wrong now you mention it! I'm half asleep this morning!

Reply:
this is reverted now, it was never published, but if you're using this module via a github ref you'll want to update

Reply:
lol